server/src/main/java/com/objectcomputing/checkins/services/checkin_notes/CheckinNoteServicesImpl.java

```diff
@@ -48,6 +46,7 @@ public CheckinNote save(CheckinNote checkinNote) {
         Boolean isAdmin = securityService != null ? securityService.hasRole(RoleType.Constants.ADMIN_ROLE) : false;
 
         if (checkinNote != null) {
+            LOG.info("Making multiple calls within this I/O thread as thread switching is costly.");
```

Comment:
These logs were important in the draft PR to demonstrate that the threading worked but we don't need them on develop

Reply:
removed the LOG.